omod/pom.xml

```diff
@@ -67,6 +79,13 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>webservices.rest-omod-common</artifactId>
 		</dependency>
+       	
+		<dependency>
+	        <groupId>org.javassist</groupId>
```

Comment:
Do both versions work with this version of javassist? I would assume you would define different javassist dependencies inside the specific profiles